### PR TITLE
Add 11ty, Scully, Sculpin, Sapper under static site generators

### LINKED ISSUE #191: Add  11ty, Scully, Sculpin, Sapper under static site generators
Add  11ty, Scully, Sculpin, Sapper under static site generators

Join the **Discord Server** for further discussions.

Server Link: https://discord.gg/HHMs7Eg

### PR DESCRIPTION
**Pull request by vivek32ta**
Fixes #191 

I have added the respective logos and linked the corresponding sites. Thank you for letting me work on this issue :)

Best,
Vivek T A

### CODE REVIEW

File: src/constants/skills.js [hunk 1]
```diff
@@ -368,7 +379,15 @@ const icons = {
...
     "https://www.vectorlogo.zone/logos/elastic/elastic-icon.svg",
   circleci:
-    "https://www.vectorlogo.zone/logos/circleci/circleci-icon.svg"
+    "https://www.vectorlogo.zone/logos/circleci/circleci-icon.svg",
+  scully:
+    "https://raw.githubusercontent.com/scullyio/scully/main/assets/logos/SVG/scullyio-icon.svg",
+  "11ty":
+    "https://www.11ty.dev/img/avatars/twitter/d4dc83e-90.webp",
+  sculpin:
+    "https://sculpin.io/css/sculpin-mascot-jackson-medium.png",
```
Comment (g-savitha) [inline src/constants/skills.js:388]: please only add svg images

**Comment by g-savitha** [comment]
Replace png image with svg. Rest everything looks fine @vivek32ta

File: src/constants/skills.js (hunk 1 above)
Comment (vivek32ta) [reply src/constants/skills.js:388]: Oh I couldn't find any svg images on their sites. 

Is there a platform you would want me to host it at? Or is a gist cool?

**Comment by vivek32ta** [comment]
I've changed the images to SVG by hosting them on a public gist.

Let me know if you want any more changes!

**Review by rahuldkjain — APPROVED** [review]
Superb.

**Pull request merged**